src/time.cr

```diff
@@ -1,66 +1,206 @@
 require "crystal/system/time"
 
-# `Time` represents an instance in incremental time. Here are some examples:
+# `Time` represents a date-time instant in incremental time observed in
```

Comment:
It's not incremental time, that guarantee is reserved for `Time.monotonic`. I'd prefer the wording "an instant in time observed in a specific time zone".

Reply:
`Time` has an incremental time-line, which does not necessarily mean a monotonic clock. In fact, it has little to do with that.

"Incremental" means each instance represent a specific moment on the time-line, as opposed to "floating time" which has no time-line at all, or field-based date-time values which can be ambiguous.

Should I add some reference, for example this link: https://www.w3.org/International/articles/definitions-time/#incremental_time

Reply:
That link should be in the type docs. I think most of us have no idea what incremental time is.